libs/set-algebra/CHANGELOG.md

```diff
@@ -14,11 +14,11 @@
 
 ## 1.1.0.1
 
-- Set upper bound ansi-wl-pprint < 1.0
+* Set upper bound ansi-wl-pprint \< 1.0
```

Comment:
I'm guessing this could be interpreted as a HTML tag if it's not escaped?

Reply:
I think it would be a safe to require any math that needs `<` to be  to encapsulated with backticks, eg:
```diff
-- Set upper bound ansi-wl-pprint < 1.0
+- Set upper bound `ansi-wl-pprint < 1.0`
```
This, I suspect would prevent `libcommonmark` from adding an escape to it.